README.md

```diff
@@ -1 +1,24 @@
 #dyscover
+
```

Comment:
you can format this document better - take a look at this https://github.com/browserstack/api#getting-available-browsers for example